app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.scss

```
@@ -2,12 +2,12 @@
 
 .document-capture-selfie-feedback {
   color: color('white');
-  background: color('black');
+  background: rgba(0, 0, 0, .5);
   left: 50%;
   top: 10%;
   position: fixed;
   transform: translateX(-50%);
-  border-radius: 5px;
-  padding: 10px;
+  padding: 0 5px 0 5px;
+  opacity: 100%;
   z-index: 11;
-}
+}
```

Nit: I think we typically include final newline, and would help avoid future changes from unrelated pull requests.

```suggestion
}

```

---

✅ [commit](https://github.com/18F/identity-idp/pull/10274/commits/b3550f7e54b8cbd006df1054eeb4671dcf597b07).